src/utils/bq_utils.py

```diff
@@ -0,0 +1,164 @@
+"""

```

Comment:
Can you refer to this package as a dependency instead of copying the implementation into our codebase?

https://github.com/SohierDane/BigQuery_Helper

Reply:
now referring that package